client/src/index.jsx

```diff
@@ -5,6 +5,12 @@ import App from './components/App';
 let listingId = window.location.href.split('/')[4];
 if (!listingId || listingId.length !== 8) {
   listingId = '11111142';
-}
+};
 
-ReactDOM.render(<App roomId={listingId} />, document.getElementById('root'));
+const Neighborhoods = () => (
+  <App roomId={listingId} />
+);
+
+// ReactDOM.render(<App roomId={listingId} />, document.getElementById('root'));
```

Comment:
We can safely remove this commented out code now

Reply:
Actually, talk to Landon about the group decision on how to handle this line. We may want to uncomment it and let the rendering happen here, instead of in the proxy server. Either way works, but we all must be consistent and do the same thing for the final product to work.